sys/include/net/gnrc/ipv6.h

```diff
@@ -11,10 +11,63 @@
  * @ingroup     net_gnrc
  * @brief       GNRC's IPv6 implementation
  *
- * The IPv6 control thread understands messages of type
- *
- *  * @ref GNRC_NETAPI_MSG_TYPE_RCV, and
- *  * @ref GNRC_NETAPI_MSG_TYPE_SND,
+ * This module is for usage with the @ref net_gnrc_netapi
+ *
+ * # NETAPI command documentation
```

Comment:
shouldn't this be `IPv6 NETAPI command documentation`?

Reply:
Why should it?

Reply:
Ping @smlng?

Reply:
bc it's in the IPv6 header file and not in `sys/include/net/gnrc/netapi.h`. If this is actually the NETAPI documentation it should go into that file and not here ... at least that seems to be sensible to me?

Reply:
But that's exactly why I think it doesn't need more specification. This is the documentation of netapi commands that IPv6 accepts (not netapi documentation). The fact that this is IPv6-related is clear through the fact that it is in this file / group. Or does it really seem unclear for you in [the 6LoWPAN equivalent](http://doc.riot-os.org/group__net__gnrc__sixlowpan.html) (which is up for years already).

Reply:
How about rewording the heading to something less ambiguous? Maybe...
```suggestion
 * # Supported NETAPI commands
```

Reply:
@miri64 ah okay, I think I get your point now ... I wasn't aware of the 6lo doc (at least that it is rather similar). However, what I saw was that the NETAPI module does not have such (extensive) documentation, hence I thought it might be misplaced here and should go in that header. 

TL;DR: leave as here, but might be nice to add some generic documentation/text on NETAPI in the respective header (later), too.

Reply:
On another note: @Lotterleben makes sense to me anyway.

Reply:
> TL;DR: leave as here, but might be nice to add some generic documentation/text on NETAPI in the respective header (later), too.

[The generic GNRC doc](http://doc.riot-os.org/group__net__gnrc.html) contains a lot about NETAPI. Since NETAPI is a pretty central part to GNRC, I don't know how much sense it makes to redundantly add extensive documentation to NETAPI then.

> On another note: @Lotterleben's comment makes sense to me anyway.

Since that also requires a change to 6LoWPAN to keep them in sync, I rather would vote to do that in a separate PR. How about that?

Reply:
sounds good to me.

Reply:
> Since that also requires a change to 6LoWPAN to keep them in sync, I rather would vote to do that in a separate PR. How about that?

See https://github.com/RIOT-OS/RIOT/pull/10718